arrows/core/applets/dump_klv.cxx

```diff
@@ -11,18 +11,27 @@
 #include <iostream>
 #include <fstream>
 
+
```

Comment:
Ack, how/when did this sneak in and not get spotted before now?
```suggestion
```

Reply:
No idea, I went back a handful of commits and didn't see it introduced. Addressed.